app/Http/Controllers/ReceiptController.php

```diff
@@ -137,6 +139,31 @@ public function storeReceipt(StoreReceiptRequest $request)
         StoreReceiptItems::run($request->item, $request->quantity, $reference->id);
         UpdateInventoryItemQuantity::run($request->item, $request->quantity, 'decrease');
 
+        for ($i=0; $i < count($request->item); $i++) {
+        $inventory = Inventory::where('id', $request->item[$i]->value)->first();
+        // Create notification if inventory quantity has reached 0
+        if($inventory->quantity == 0 && $inventory->notify_critical_quantity == 'Yes'){
+            Notification::create([
+                'reference_id' => $inventory->id,
+                'message' => 'Inventory item '.$inventory->item_name.' has zero stocks. Please reorder.',
+                'title' => 'Inventory Zero Stocks',
+                'type' => 'danger',
+                'link' => 'inventory/'.$inventory->id,
+            ]);
+        }
+        // Create notification if inventory quantity is less than or equal to critical level
+        else if($inventory->quantity <= $inventory->critical_quantity && $inventory->notify_critical_quantity == 'Yes'){
+            Notification::create([
+                'reference_id' => $inventory->id,
+                'message' => 'Inventory item '.$inventory->item_name.' is less than or equal to '.$inventory->critical_quantity.'. Please reorder.',
+                'title' => 'Inventory Critical Level',
+                'type' => 'warning',
+                'link' => 'inventory/'.$inventory->id,
```

Comment:
I think this should redirect instead to `Vendor > Bills`, then open bill modal and add the item with critical number of stocks. Same goes to line # 151